package.json

```
@@ -13,34 +13,59 @@
   "unpkg": "dist/ionicons.js",
   "collection": "dist/collection/collection-manifest.json",
   "collection:main": "dist/collection/index.js",
+  "exports": {
+    ".": {
+      "types": "./dist/types/index.d.ts",
+      "default": "./dist/index.js",
+      "require": "./dist/index.cjs.js"
+    },
+    "./components": {
+      "types": "./dist/components/index.d.ts",
+      "import": "./dist/components/index.js"
+    },
+    "./icons": {
+      "types": "./icons/index.d.ts",
+      "import": "./icons/index.mjs",
+      "require": "./icons/index.js"
+    }
+  },
   "scripts": {
-    "compile.scripts": "tsc -p scripts/tsconfig.json",
-    "build": "npm run compile.scripts && npm run build.files && npm run build.component && npm run collection.copy",
-    "build.files": "node scripts/build.js",
+    "build": "run-s clean build.* prettier",
+    "build.files": "tsx scripts/build.js",
     "build.component": "stencil build",
-    "collection.copy": "node scripts/collection-copy.js",
-    "start": "npm run compile.scripts && npm run build.files && stencil build --dev --watch --serve",
-    "test": "npm run test.spec",
-    "test.spec": "stencil test --spec"
+    "build.collection": "tsx scripts/collection-copy.js",
```

Can we get rid of `build.collection`? It doesn't seem to be used anywhere. If we still need it then should it be?:

```suggestion
    "build.collection": "tsx scripts/collection-copy.ts",
```

If the command is needed, then let's add a comment on `collection-copy` explaining what the purpose of the script is. It's not clear to me what it's for.